scripts/src/main/resources/scripts/command/ide

```diff
@@ -44,27 +44,6 @@ function doUpdateMetaFile() {
   fi
 }
 
-function doUpdateScripts() {
-  # shellcheck disable=SC2007,SC2154
-  doUpgradeMavenArtifact "${DEVON_IDE_HOME}" "${DEVON_IDE_REPO_URL}" "devonfw-ide-scripts" "${target_version}" ".tar.gz" "$[devon_ide_version]"
-  if [ "${?}" = 0 ]
-  then
-    if [ "${target_version}" = "LATEST" ]
-    then
-      cp "${DEVON_IDE_HOME}/scripts/devon" ~/.devon/devon
-    fi
-    if doIsWindows
-    then
-      if [ "${target_version}" = "LATEST" ]
-      then
-        cp "${DEVON_IDE_HOME}/scripts/devon.bat" "${USERPROFILE}/scripts/devon.bat"
-      fi
-      doWarning "To prevent windows file locking errors, we update async in background now.\nPlease wait a minute and then rerun 'devon ide update' to complete."
-      exit 0
-    fi
-  fi
-}
```

Comment:
This function is called in multiple places.
As you removed it, you broke the code as there are remaining places where it is still called that will stop working.